src/en/foundations/spacing/tokens.md

```diff
@@ -2,11 +2,16 @@
 title: Spacing - Foundations
 layout: "layouts/token-documentation.njk"
 translationKey: "spacingTokens"
-tags: ['spacingEN', 'tokens']
+tags: ["spacingEN", "tokens"]
+type: dimension
```

Comment:
Would you be able to scope any frontmatter information for the token tables into an object so it's easy to tell what it's for?

Reply:
I think I can do that. Do you mean grouping type, useCases, tableHeaders and such into something like a `tokenTable` object

Reply:
Exactly!

Reply:
I was able to do it, but we have to explicitly pass the type along with the partial. I think its a bit clearer so all good!